1 table pet grooming biz scenario.sql

```diff
@@ -0,0 +1,34 @@
+
+use master
+drop database if exists PetGroomingDB
+GO
+create DATABASE PetGroomingDB
+Go
+use PetGroomingDB
+
+drop table if EXISTS dbo.Petgroom
+go
+create table dbo.Petgroom(
+PetGroomID Int not null identity primary key,
+PetOwnerFirstName varchar(30) not null 
+CONSTRAINT c_PetGroom_Pet_Owner_First_Name_cannot_be_blank CHECK(PetOwnerFirstName <> ''),
+PetOwnerLastName varchar(30) not null
+CONSTRAINT c_PetGroom_Pet_Owner_Last_Name_cannot_be_blank CHECK(PetOwnerLastName <> ''),
+PetOwnerAddress varchar(100) not null
+CONSTRAINT c_PetGroom_Pet_Owner_Address_cannot_be_blank CHECK(PetOwnerAddress <> ''),
+PetType varchar(10) not null
+CONSTRAINT c_PetGroom_Pet_Type_can_only_be_dog_cat_rabbit_or_guinea_pig CHECK(PetType in ('dog', 'cat', 'rabbit', 'guinea pig')),
+PetName varchar(30) not null 
+CONSTRAINT c_PetGroom_Pet_Name_cannot_be_blank CHECK(PetName <> ''),
+PricePerGrooming decimal(4,2) not null,
```

Comment:
AS: Add a constraint that cost must be greater than zero, as to not let negative numbers.